P: ampparit.com

peace2000 opened the pull request:
Cookies need to be accepted, otherwise it's not possible to use 3rd-party providers to login to the page.

![kuva](https://github.com/easylist/easylist/assets/17256841/3faf9918-864e-4011-9a6c-e02ce7c57cd0)

![kuva](https://github.com/easylist/easylist/assets/17256841/27972718-adf3-41af-a6a3-e934cfb3bbde)


`Evästeasetuksesi estävät kirjautumisen käyttäen Facebookia. Ole hyvä ja tarkasta evästeasetuksesi.` =
`Your cookie settings prevent you from logging in by using Facebook. Please check your cookie settings.`

Handled by uAssets: https://github.com/uBlockOrigin/uAssets/commit/ab3bf04d4b3a97fd4491bc6e4225ba8d43003185

Khrin commented:
In view of its many exceptions, is there still a need for this filter?
`||almamedia.fi/almacmp/$script,from=~arvopaperi.fi|~iltalehti.fi|~mediuutiset.fi|~mikrobitti.fi|~talouselama.fi|~tekniikkatalous.fi|~tivi.fi|~www.kauppalehti.fi|~www.uusisuomi.fi`

peace2000 commented:
@Khrin Yeah, you are right. It's kinda pointless rule so I removed it.

Also adjusted:
www.kauppalehti.fi -> kauppalehti.fi
www.uusisuomi.fi -> uusisuomi.fi

So that googletagmanager is whitelisted for their subdomains also. I didn't notice in the past that their subdomains need to be have GDPR whitelisted also. (E.G. : https://keskustelu.kauppalehti.fi/ & https://puheenvuoro.uusisuomi.fi/
